rpcs3/Emu/Cell/lv2/sys_net/lv2_socket_p2ps.cpp

```diff
@@ -502,6 +540,36 @@ void lv2_socket_p2ps::send_u2s_packet(std::vector<u8> data, const ::sockaddr_in*
 	}
 }
 
+void lv2_socket_p2ps::close_stream_nl(nt_p2p_port* p2p_port)
+{
```

Comment:
No param check?

Reply:
No, I only use a pointer because of circular dependencies and I couldn't use a reference, it can't be nullptr.